### PR TITLE
[CMSIS-NN] Separated symmetric and asymmetric padding tests for Conv2D

### PR DESCRIPTION
**Pull request by ashutosh-arm**
* Separated asymmetric padding test from others.
  This is a follow up from https://github.com/apache/tvm/pull/9886.

### CODE REVIEW

**Comment by ashutosh-arm** [comment]
CC:  @Mousius @grant-arm for code review.

File: tests/python/contrib/test_cmsisnn/test_conv2d.py [hunk 1]
```diff
@@ -124,7 +124,113 @@ def make_model(
 
 
 @tvm.testing.requires_cmsisnn
-@pytest.mark.parametrize("ifm_shape", [(1, 25, 25, 12), (1, 64, 100, 4)])
+@pytest.mark.parametrize("ifm_shape", [(1, 64, 100, 4)])
+@pytest.mark.parametrize("kernel_size", [(3, 3)])
+@pytest.mark.parametrize("padding", ["SAME", "VALID"])
+@pytest.mark.parametrize("strides, dilation", [((1, 1), (1, 1))])
+@pytest.mark.parametrize("relu_type", ["RELU"])
```
Comment (Mousius) [inline tests/python/contrib/test_cmsisnn/test_conv2d.py:131]: Make these variables instead of parametrize so it's clear they don't change.

**Review by Mousius — CHANGES_REQUESTED** [review]

File: tests/python/contrib/test_cmsisnn/test_conv2d.py (hunk 1 above)
Comment (ashutosh-arm) [reply tests/python/contrib/test_cmsisnn/test_conv2d.py:128]: Made the change. Let's see how the checks go.

**Review by Mousius — APPROVED** [review]
LGTM! Thanks @ashutosh-arm!

**Pull request merged**